### ISSUE TITLE
Please consider configuring rembg for Gimp (it doesn't reduce quality and works 100% offline)

### ISSUE THREAD

**Issue by superdreamkilla** [manu12121999/RemoveBG-GIMP#5]
Many people who use Gimp  on Linux  do not like the concept of using a online tool that cannot be used locally offline, as they value privacy. You may not have known but https://github.com/danielgatis/rembg is a open source python neural network that works 100% offline and can be configured natively against ones system or a static miniconda environment profile.  This allows the user to remove image backgrounds in high quality without sending their data to a foreign server.

please consider configuring rembg with Gimp. Below is a thread about it and some test I ran. I even made a bash script for myself to activate rembg from my start menu in a desired folder.

https://forums.linuxmint.com/viewtopic.php?f=42&t=339124

**TEST**
![image](https://user-images.githubusercontent.com/61440491/105648957-0cf69200-5ea6-11eb-8c20-58428176bd2c.png)

![image](https://user-images.githubusercontent.com/61440491/105648966-1253dc80-5ea6-11eb-84a5-75097709f136.png)

![image](https://user-images.githubusercontent.com/61440491/105648996-344d5f00-5ea6-11eb-8972-67403bdfaa13.png)

**Comment by manu12121999** [comment]
Nice, looks promising. 
I was searching for an open source alternative to remove.bg (I have even begun to train a neural network by myself for this purpose), but I did not find this repo. I will definitely look into it and maybe integrating it into GIMP (but not in the next two or three months). 

Anyway, thank you for mentioning it.

**Comment by superdreamkilla** [comment]
yes,  Thank you for consider.ing. It would be preferable if it worked with versions of rembg that run in isolated python environments and it will need to give the user an option to enter the local path URL to rembg.

**Comment by raniaamina** [comment]
rembg is awesome, currently i try to create the gui for this tool, hope it can help also

**Comment by Tarek-Hasan** [comment]
There is also now [DIS](https://github.com/xuebinqin/DIS) (v2 upcoming), Highly Accurate Dichotomous Image Segmentation from the maker of [U2-Net](https://github.com/xuebinqin/U-2-Net).
